### ISSUE TITLE
Different types of biomes in one world 

### ISSUE THREAD

**Issue by kasperdun** [neki-dev/gen-biome#1]
Привет, текущая реализация не генерирует разные биомы, она создает один биом с разными высотами, биомы как я понимаю подразумевают разный тип местности, например я хочу создать мир в котором будут залежи различных ресурсов отдельными друг от друга как в факторио, но получается только мир с залежами в которых сразу все типы ресурсов.

Пример:
   ```
 const BIOMES = [
      {
        params: { lowerBound: 0.0, upperBound: 0.2 },
        data: { type: CellType.Coal },
      },
      {
        params: { lowerBound: 0.2, upperBound: 0.4 },
        data: { type: CellType.CopperOre },
      },
      {
        params: { lowerBound: 0.4, upperBound: 0.6 },
        data: { type: CellType.IronOre },
      },
      {
        params: { lowerBound: 0.6 },
        data: { type: CellType.Grass },
      },
    ];
```

На выходе мир где все ресурсы друг на друге, а нужны разные ресурсы разбросанные по миру. 
Может уже есть возможность так сделать, просто я не нашел как?

**Comment by neki-dev** [comment]
Привет! Возможно я плохо интерпретировал название массива `BIOMES`, по-факту это слои карты. 

Но в текущей реализации так же можно реализовать то, что Вам требуется путем раздельных генераций (для каждого биома своя генерация мира).

Написал минимальный пример: https://codesandbox.io/p/sandbox/javascript-forked-lntj2q

Решение выглядит не совсем удобным, по-этому планирую добавить более элегантное решение из коробки в ближайшее время.

**Comment by kasperdun** [comment]
Верно, в текущей реализации BIOMES можно переименовать в LAYERS, и добавить BIOMES - только нужно подумать что это будет
Мб что-то такое

```
BIOMES = MAP<string, Layers>

BIOMES.set('CoalBiome', [... layers ...])
```

мб получится удобнее придумать
